Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -213,6 +213,12 @@ final class PledgePaymentMethodsViewController: UIViewController {
       self.tableView.reloadSections([PaymentMethodsTableViewSection.addNewCard.rawValue], with: .none)
     }
   }
+
+  // MARK: - Public functions
+
+  func setShouldCancelPaymentSheetAppearance(hidden: Bool) {
+    self.viewModel.inputs.shouldCancelPaymentSheetAppearance(state: hidden)
```

Comment:
Ah, or is this more like a dismissal?

Reply:
I think this name is more clear than it was before but you're right - when I named it, I didn't know what it did either 😭. You're pretty much correct, though. Basically when you add a new card to the payment sheet, there's a loading indicator for a bit before the sheet pops up. Setting this to false (the only way it's used outside of tests) cancels that pop up (usually because we've presented a different modal instead). I _think_ it might also dismiss the sheet itself if it's already presented by the time this is called, but I'm not sure about that one since that would only happen as a race condition. If you have any ideas for a different name, though, let me know!

Reply:
Hmmm...what about something like `dontShowPaymentSheet`?

Reply:
I'm just going to name it `cancelModalPresentation` I think. That's clear enough while still being general enough that you don't need to understand the paymentMethodsVC to understand what the goal is.